### ISSUE TITLE
CIFAR10 accuracy

### ISSUE THREAD

**Issue by MrChenFeng** [Spijkervet/SimCLR#12]
Hi Spijkervet,@Spijkervet 

have you tried more epochs for your experiment settings, like 500 epochs,
the result from the paper has shown that even 256 batchsize could achieve about 93% accuracy for 500 epochs traning.
However, my implementation can only got ~89.5% accuracy even with 512 batchsize.

Best,
Chen

**Comment by gauenk** [comment]
I also don't get the reported accuracy. Even with the code and pretrained weights from @Spijkervet I do not get the reported test accuracy claimed in this repo: only 75% compared to the reported 83% for CIFAR10. I wonder if the lars optimizer has something to do with it. I have really tried to get it working but I end up with "nan" values for my training loss.

**Comment by LanXiaoPang613** [comment]
Hi chen, if you used the code which is shared by Spijkervet? Why i only achieved nearly 65% acc on CIFAR-10. It seems that used 128 batch size train the representation may lead to acc decrease.

**Comment by MrChenFeng** [comment]
> Hi chen, if you used the code which is shared by Spijkervet? Why i only achieved nearly 65% acc on CIFAR-10. It seems that used 128 batch size train the representation may lead to acc decrease.

I havent tried Spijkervet implementation. In my implementations, epoch num is more important compared to batchsize. However, I cannot get better result than 90% with resnet18 in all settings.
